### ISSUE TITLE
Feedback about external references not being available on File SCOs

### ISSUE THREAD

**Issue by clenk** [oasis-tcs/cti-stix2#317]
The python-stix2 repo received some feedback about `external_references` not being available on the `File` object: https://github.com/oasis-open/cti-python-stix2/issues/568

> Hi,
> 
> What a poor decision invalidating External References for File objects. What about the situation where i have an external BLOB store that i want to connect to my knowledge graph???? How am i meant to store file objects in a knowledge graph????
> 
> Based on an arbitrary decision to stop External References for Files (like why????), I cannot support this use case, and have to either reintroduce an illegal property, or use the Artifact object, which has very poorly thought through mechanisms for description
> 
> For me to store the file as an artifact, and use the url field, i must hash the file. Worse, I can't name the file or provide a description of it, since there are no fields for this in the Artifact object. So in the case where i want to store a text file, it is a nameless url, while it has to be hashed I am unable to describe it, which is frankly weak for most apparent use cases.
> 
> This seems a set of very poor decision without any real rationale. I am not happy about this at all. You make it very hard to utilise Stix 2.1. Sorry if I am over-passionate about this, but it is a blockage for me

**Comment by brettforbes** [comment]
yeah, sorry i allowed my frustrations to boil over there, but it is annoying. It is actually easy for me to add external references only within my own system for the File object, but it makes it non-standard, which i hate. I am really not sure what to do here, since the Artifcat option is just as ugly since i cant know anything about the file then (like no name, no description etc). What to do?
